File: Beecrowd/Beginner/1435 - 1985/1789 - The Race of Slugs/C++/1789_TheRaceofSlugs.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int slugs;

    while (cin >> slugs) {
        int maxSpeed = 0;

        for (int i = 0; i < slugs; i++) {
            int speed;
            cin >> speed;

            if (speed > maxSpeed) maxSpeed = speed;
        }

        if (maxSpeed < 10) cout << 1 << endl;
        else if (maxSpeed >= 10 && maxSpeed < 20) cout << 2 << endl;
        else cout << 3 << endl;
    }

    return 0;
}

File: Beecrowd/Beginner/1435 - 1985/1789 - The Race of Slugs/Python/1789_TheRaceofSlugs.py
while True:
    try:
        slugs = int(input())
        speed = input().split()
        max_speed = 0

        for i in range(slugs):
            if max_speed < int(speed[i]):
                max_speed = int(speed[i])

        if max_speed < 10:
            print(1)
        elif 10 <= max_speed < 20:
            print(2)
        else:
            print(3)

    except EOFError:
        break
